openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, course_key, request, username=''):
         )
         self.effective_user = self.overview.effective_user
         self.course_key = course_key
+        self.enrollment_object = CourseEnrollment.get_enrollment(self.effective_user, self.course_key,
```

Comment:
Why `enrollment_object` vs just `enrollment`? I think I might prefer `enrollment_object` here, but curious why you went to this one

Reply:
Because `enrollment` is already an `@property` on the class and it returns a dict like `{'mode': mode, 'is_active': is_active}`, not the actual enrollment object. Thus `enrollment_object` to differentiate.

Reply:
Ah, right. Forgot about that property